src/developTest/java/mServer/crawler/AddToFilmlistTest.java

```diff
@@ -0,0 +1,131 @@
+                    case "/" + FILM_NAME_ONLINE2:
+                        return new MockResponse().setResponseCode(200);
+                    default:
+                        return new MockResponse().setResponseCode(404);
+                }
+            }
+        };
+        mockServer.setDispatcher(dispatcher); 
+        mockServer.start();
+        baseUrl = mockServer.url("").toString();
+    }
+    
+    @AfterClass
+    public static void teardownClass() throws IOException {
+        mockServer.shutdown();
+    }
+
+    @Before
+    public void setUp() {
+        listToAdd = new ListeFilme();
+        list = new ListeFilme();
+        list.add(createTestFilm(Const.BR, FILM_TOPIC1, FILM_TITLE1, "film1.mp4"));
+        list.add(createTestFilm(Const.BR, FILM_TOPIC2, FILM_TITLE1, "film2.mp4"));
+    }
+
+    @Test
+    public void testAddOldListDifferentSenderAndUrlAdded() {
+        listToAdd.add(createTestFilm(Const.ARD, FILM_TOPIC1, FILM_TITLE1, FILM_NAME_ONLINE));
+        
+        AddToFilmlist target = new AddToFilmlist(list, listToAdd);
+        target.addOldList();
+        
+        assertThat(list.size(), equalTo(3));
+    }
+    
+    @Test
+    public void testAddOldListDifferentTopicAndUrlAdded() {
+        listToAdd.add(createTestFilm(Const.BR, FILM_TOPIC3, FILM_TITLE1, FILM_NAME_ONLINE));
+        
+        AddToFilmlist target = new AddToFilmlist(list, listToAdd);
+        target.addOldList();
+        
+        assertThat(list.size(), equalTo(3));
+    }
+    
+    @Test
+    public void testAddOldListDifferentTitleAndUrlAdded() {
+        listToAdd.add(createTestFilm(Const.BR, FILM_TOPIC1, FILM_TITLE2, FILM_NAME_ONLINE));
+        
+        AddToFilmlist target = new AddToFilmlist(list, listToAdd);
+        target.addOldList();
+        
+        assertThat(list.size(), equalTo(3));
+    }
+    
+    @Test
+    public void testAddOldListDifferentUrlNotAdded() {
+        listToAdd.add(createTestFilm(Const.BR, FILM_TOPIC1, FILM_TITLE1, FILM_NAME_ONLINE2));
+        
+        AddToFilmlist target = new AddToFilmlist(list, listToAdd);
+        target.addOldList();
+        
+        assertThat(list.size(), equalTo(2));
+    }
+    
+    @Test
+    public void testAddOldListDifferentTitleAdded() {
+        listToAdd.add(createTestFilm(Const.BR, FILM_TOPIC1, FILM_TITLE3, FILM_NAME_ONLINE));
+        
+        AddToFilmlist target = new AddToFilmlist(list, listToAdd);
+        target.addOldList();
+        
+        assertThat(list.size(), equalTo(3));
+    }
+    
+    @Test
+    public void testAddOldListDifferentTitleAndUrlButNotOnlineNotAdded() {
+        listToAdd.add(createTestFilm(Const.BR, FILM_TOPIC1, FILM_TITLE2, "imnotonline.mp4"));
+        
+        AddToFilmlist target = new AddToFilmlist(list, listToAdd);
+        target.addOldList();
+        
+        assertThat(list.size(), equalTo(2));
+    }
+    
+    private static DatenFilm createTestFilm(String sender, String topic, String title, String filmUrl) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "createTestFilm" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)